### PR TITLE
[redesign] Results page

### LINKED ISSUE #75: [redesign] Results cards and results page
Implement the new results cards and results page as per [specs](https://xd.adobe.com/spec/2af300cc-83cf-4951-5fc8-85f2e3d389e3-5cc2/screen/9a53a3bf-63b2-4256-a6be-4bad9fb38458/Result-5)

### PR DESCRIPTION
**Pull request by thi4go**
This PR implements design specs related to the results page, and refactors some of the existing code. 

edit: Mobile version can be reviewed in this PR also. :+1: 

~~merged - build on top of #91~~
part of #85 
closes #75 #54 #84

### CODE REVIEW

File: src/components/Results/ResultCard.jsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { Card, Text, Tooltip } from "pi-ui";
+import CopyIcon from "src/assets/copy_icon.svg";
+import FileIcon from "src/assets/file_icon.svg";
+import HashIcon from "src/assets/hash_icon.svg";
+import DownloadIcon from "src/assets/download_icon.svg";
+import styles from "./Results.module.css";
+
+const ResultCard = ({
```
Comment (tiagoalvesdulce) [inline src/components/Results/ResultCard.jsx]: Basically, this component is too big and difficult to maintain and should be broken down into smaller components. You can create Name, Digest, ChainInfo and DownloadProof components, for example.

File: src/components/Results/hooks.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  timestamp,
+  verify
+} from "src/helpers/dcrtime";
+
+const useProcessDigests = location => {
+  const [digests, setDigests] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
```
Comment (tiagoalvesdulce) [inline src/components/Results/hooks.js]: this useEffect hook is too big. Consider breaking it into smaller functions.

File: src/components/Results/hooks.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  return { digests, loading, error };
+};
+
+const updateDigests = (digests, newDigests) =>
+  digests.map(d => {
+    const newDigest = newDigests.filter(nd => nd.digest === d.digest)[0];
+    return newDigest || d;
+  });
+
+export default useProcessDigests;
```
Comment (tiagoalvesdulce) [inline src/components/Results/hooks.js]: This code doesn't belong to the hooks file.

**Comment by thi4go** [comment]
Hey @tiagoalvesdulce, thanks for the review and suggestions, my last commit addresses all your comments :+1:

**Pull request merged**